macros/get_columns_from_sql.sql

```diff
@@ -0,0 +1,31 @@
+{#-
+This query returns a list of column names. This is particularly useful when you
+need the column names from a CTE (e.g. an ephemeral model), rather than a relation
+(i.e. a view or table) since the
+Note that it only returns column_names, and not a Column object
+-#}
+{%- macro get_columns_from_sql(query_sql) -%}
```

Comment:
does this file need to be here? there's a similar macro in dbt-core (new in 0.14.0, i think) which implements this: https://github.com/fishtown-analytics/dbt/blob/dev/louisa-may-alcott/core/dbt/include/global_project/macros/adapters/common.sql#L31-L45

Reply:
nope! I think i accidentally committed that :)